app/models/organization.rb

```diff
@@ -158,13 +158,16 @@ def law_enforcement_missing_verified_features?
   end
 
   def paid_for?(feature_name)
-    return false unless feature_name.present? && paid_feature_slugs.is_a?(Array)
-    # If kinds is an array, make sure they all are permitted kinds
-    if feature_name.is_a?(Array)
-      return false unless feature_name.any? # If they passed an empty array, it's false
-      return feature_name.none? { |k| !paid_for?(k) }
+    features =
+      Array(feature_name)
+        .map { |name| name.strip.downcase.gsub(/\s/, "_") }
+
+    return false unless features.present? && paid_feature_slugs.is_a?(Array)
+
+    features.all? do |feature|
```

Comment:
Nice. This is an excellent use of `all?`.

However - looking through the code, I don't see anywhere that we actually use an array. I think having this method accept an array is additional complexity that we can remove.

Since you're in here, would you remove the

Reply:
We do have the method accept an array elsewhere, actually:

```rb
# spec/requests/organized/messages_request_spec.rb L3-17 (a3c48cbf)

RSpec.describe Organized::MessagesController, type: :request do
  include_context :geocoder_default_location
  include_context :organization_with_geolocated_messages
  let(:base_url) { "/o/#{organization.to_param}/messages" }
  # Request specs don't have cookies so we need to stub stuff if we're in request specs
  # This is suboptimal, but hey, it gets us to request specs for now
  before { allow(User).to receive(:from_auth) { user } }

  describe "messages root" do
    let(:user) { FactoryBot.create(:organization_member, organization: organization) }
    it "renders" do
      get base_url, json_headers
      expect(response.status).to eq(200)
      expect(response).to render_template(:index)
    end
```

Reply:
This is exactly why I wish I didn't do stupid things like assign an array to a singular variable.

I guess you should merge this PR and ignore this.

I'll open a separate PR to rename this variable